recipe/meta.yaml

```
@@ -1,7 +1,7 @@
 package:
   name: conda
   # VERSION_OVERRIDE is used by the canary release workflow
-  version: {{ os.getenv("VERSION_OVERRIDE") or GIT_DESCRIBE_TAG }}.{{ GIT_BUILD_STR }}
+  version: {{ os.getenv("VERSION_OVERRIDE") or GIT_DESCRIBE_TAG + "+" + GIT_BUILD_STR }}
```

This will give version like `25.3.1.56+g1ec58ac92` for the conda package were 56 is the number of commits since the last tags and the hash after the `+` is the git hash. 

```suggestion
  version: {{ os.getenv("VERSION_OVERRIDE") or GIT_DESCRIBE_TAG + "+" + GIT_BUILD_STR }}
  version: {{ os.getenv("VERSION_OVERRIDE") or GIT_DESCRIBE_TAG }}.{{ GIT_DESCRIBE_NUMBER }}+{{ GIT_DESCRIBE_HASH }}
```